### ISSUE TITLE
SIGSEGV on Chromium (openSUSE Tumbleweed)

### ISSUE THREAD

**Issue by cyc1ingsir** [keepassxreboot/keepassxc-browser#1957]
## Expected Behavior
Adding extension to Chromium is possible.  

## Current Behavior
Chromium crashes with a SIGSEGV shortly after startup when KeepassXC extension is installed.

## Possible Solution
None known

## Steps to Reproduce (for bugs)
Running Chromium
Version 115.0.5790.170 (openSUSE Build) stable (64-bit)
on openSUSE Tumbleweed with all updates installed

Starting with a fresh profile (no other extensions are installed) `chromium --user-data-dir=/home/user/.config/new-profile`
Chromium starts.

Going to the Chrom Web Store, add KeepassXC extension.
Chromium crashes 2 seconds later.

Restarting Chromium - Chromium crashes 2 seconds after startup.
Starting with  `chromium --user-data-dir=/home/user/.config/new-profile --disable-extensions` starts Chromium again but without the freshly installed extension.
This started to happen after the latest chromium update. 

## Debug info
```
Received signal 11 SEGV_MAPERR 000000000000
#0 0x56524abad382 base::debug::CollectStackTrace()
#1 0x56524ab99653 base::debug::StackTrace::StackTrace()
#2 0x56524abace61 base::debug::(anonymous namespace)::StackDumpSignalHandler()
#3 0x7f8805c41330 (/usr/lib64/libc.so.6+0x3e32f)
#4 0x7f8805d6b6ad __strlen_avx2
#5 0x5652497bdd24 extensions::FormDataParser::CreateFromContentTypeHeader()
#6 0x5652497bd937 extensions::FormDataParser::Create()
#7 0x5652497bfdfc extensions::ParsedDataPresenter::ParsedDataPresenter()
#8 0x5652497dce6a extensions::WebRequestInfoInitParams::WebRequestInfoInitParams()
#9 0x5652497df857 extensions::WebRequestProxyingURLLoaderFactory::InProgressRequest::UpdateRequestInfo()
#10 0x5652497e5dbc extensions::WebRequestProxyingURLLoaderFactory::CreateLoaderAndStart()
#11 0x5652470be95e network::mojom::URLLoaderFactoryStubDispatch::Accept()
#12 0x56524b3dc40e mojo::InterfaceEndpointClient::HandleValidatedMessage()
#13 0x56524b3e3d6e mojo::MessageDispatcher::Accept()
#14 0x56524b3ddc6d mojo::InterfaceEndpointClient::HandleIncomingMessage()
#15 0x56524b3e7773 mojo::internal::MultiplexRouter::ProcessIncomingMessage()
#16 0x56524b3e703d mojo::internal::MultiplexRouter::Accept()
#17 0x56524b3e3dad mojo::MessageDispatcher::Accept()
#18 0x56524b3d9a2f mojo::Connector::DispatchMessage()
#19 0x56524b3da210 mojo::Connector::ReadAllAvailableMessages()
#20 0x565247a2b4dc mojo::SimpleWatcher::DiscardReadyState()
#21 0x56524b3fba6e mojo::SimpleWatcher::OnHandleReady()
#22 0x56524b3fbe02 base::internal::Invoker<>::RunOnce()
#23 0x56524ab3a4e1 base::TaskAnnotator::RunTaskImpl()
#24 0x56524ab55c52 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWorkImpl()
#25 0x56524ab55696 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork()
#26 0x56524ab56135 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::DoWork()
#27 0x56524abc038d base::MessagePumpGlib::HandleDispatch()
#28 0x56524abc0cfd base::(anonymous namespace)::WorkSourceDispatch()
#29 0x7f880868b988 g_main_context_dispatch
#30 0x7f880868bd98 (/usr/lib64/libglib-2.0.so.0.7600.4+0x5dd97)
#31 0x7f880868be2c g_main_context_iteration
#32 0x56524abc0568 base::MessagePumpGlib::Run()
#33 0x56524ab564d0 base::sequence_manager::internal::ThreadControllerWithMessagePumpImpl::Run()
#34 0x56524ab19078 base::RunLoop::Run()
#35 0x5652489fedcd content::BrowserMainLoop::RunMainMessageLoop()
#36 0x565248a00bb2 content::BrowserMainRunnerImpl::Run()
#37 0x5652489fc1d3 content::BrowserMain()
#38 0x565249f4db42 content::RunBrowserProcessMain()
#39 0x565249f4f203 content::ContentMainRunnerImpl::RunBrowser()
#40 0x565249f4f01c content::ContentMainRunnerImpl::Run()
#41 0x565249f4c770 content::RunContentProcess()
#42 0x565249f4cd59 content::ContentMain()
#43 0x565246954765 ChromeMain
#44 0x7f8805c2abf0 __libc_start_call_main
#45 0x7f8805c2acb9 __libc_start_main_alias_2
#46 0x5652464f6035 _start
  r8: 0000000000000002  r9: 0000000000000000 r10: 00005652511e3000 r11: 0000000000000065
 r12: 00007ffdcc8a01d8 r13: 000000000000002f r14: 00002868003c8140 r15: 00002868041ca850
  di: 0000000000000000  si: 0000000000000001  bp: 00007ffdcc8a00f0  bx: 00007ffdcc8a0001
  dx: 0000000000000000  ax: 0000000000000000  cx: 00007f8805df3040  sp: 00007ffdcc8a0038
  ip: 00007f8805d6b6ad efl: 0000000000010283 cgf: 002b000000000033 erf: 0000000000000004
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]

'chromium --user-data-dir=/home/…' terminated by signal SIGSEGV (Address boundary error)
```
KeePassXC - 2.7.5
KeePassXC-Browser - 1.8.7
Operating system: Linux
Browser: Chromium 115.0.5790.170

**Comment by phoerious** [comment]
You should report this to Chromium, not us. There is no way an extension can crash your browser unless there is a bug in the browser.

**Comment by cyc1ingsir** [comment]
Sorry, you are right. The KeepassXC extension was the first I tested since this is the most important one for me.

**Comment by cyc1ingsir** [comment]
You are right. The KeepassXC extension was the first I tested, since it's the most valuable for me.
However, I tested randomly a couple more extensions and whilst "react developer tools" "google translate" could be added without problems,  with "AdblockPlus", "uBlockOrigin" and "KeepassXC" I could reproduce this crash. Seems to be related to a function, these plugins are using.
Let me use this opportunity to say thank you for this valuable extension and your swift response!
